lib/pdk/cli/exec.rb

```diff
@@ -135,7 +135,8 @@ def run_process!
           begin
             @process.start
           rescue ChildProcess::LaunchError => e
-            raise PDK::CLI::FatalError, _("Failed to execute '%{command}': %{message}") % { command: argv.join(' '), message: e.message }
+            raise PDK::CLI::FatalError, _('Failed to execute process: %{message}') % { message: e.message } unless @process.respond_to?(:argv)
+            raise PDK::CLI::FatalError, _("Failed to execute '%{command}': %{message}") % { command: @process.argv.join(' '), message: e.message }
```

Comment:
ugh, yeah, the GuardClause cop is a little bit hit and miss. Maybe the better alternative would be to
```
message = if @process.respond_to?(:argv)
  # ...
raise PDK::CLI::FatalError, message
```